include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -873,15 +873,14 @@ FUNCTION(GetObjCClassFromMetadata, swift_getObjCClassFromMetadata, DefaultCC,
          ARGS(TypeMetadataPtrTy),
          ATTRS(NoUnwind, ReadNone))
 
-// Metadata *swift_getTupleTypeMetadata(size_t numElements,
+// Metadata *swift_getTupleTypeMetadata(TupleTypeFlags flags,
 //                                      Metadata * const *elts,
 //                                      const char *labels,
-//                                      uint32_t flags,
 //                                      value_witness_table_t *proposed);
 FUNCTION(GetTupleMetadata, swift_getTupleTypeMetadata, DefaultCC,
          RETURNS(TypeMetadataPtrTy),
-         ARGS(SizeTy, TypeMetadataPtrTy->getPointerTo(0),
-              Int8PtrTy, Int32Ty, WitnessTablePtrTy),
+         ARGS(Int32Ty, TypeMetadataPtrTy->getPointerTo(0),
```

Comment:
The reason that these functions are generally declared as taking a size_t/uintptr_t is to eliminate potential ABI issues around extending the smaller type to fit the full register.  Please imitate that here (and above in the declaration of the TupleTypeFlags typedef).

Reply:
Okay!